packages/expo-google-sign-in/android/build.gradle

```diff
@@ -0,0 +1,75 @@
+
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.1.3'
+    }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '1.0.0'
+
+// Upload android library to maven with javadoc and android sources
+configurations {
+    deployerJars
+}
+
+// Creating sources with comments
+task androidSourcesJar(type: Jar) {
+    classifier = 'sources'
+    from android.sourceSets.main.java.srcDirs
+}
+
+// Put the androidSources and javadoc to the artifacts
+artifacts {
+    archives androidSourcesJar
+}
+
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            configuration = configurations.deployerJars
+            repository(url: mavenLocal().url)
+        }
+    }
+}
+
+android {
+    compileSdkVersion 26
+
+    defaultConfig {
+        minSdkVersion 19
+        targetSdkVersion 26
+        versionCode 4
+        versionName "1.0.0"
+    }
+    lintOptions {
+        abortOnError false
+    }
+}
+
+if (new File(rootProject.projectDir.parentFile, 'pubspec.yaml').exists()) {
+    apply from: project(":expo_core").file("../expo-core.gradle")
+} else if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
+    apply from: project(":expo-core").file("../expo-core.gradle")
+} else {
+    throw new GradleException(
+            "'expo-core.gradle' was not found in the usual Flutter or React Native dependency locations. " +
+            "This package can only be used in such projects. Are you sure you've installed the dependencies properly?")
+}
+
+dependencies {
+    expendency "expo-core"
+    api 'com.google.android.gms:play-services-auth:15.+'
```

Comment:
Gradle will be sad that you haven't specified a proper version of `play-services-auth`. 😃